src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx

```diff
@@ -94,8 +104,15 @@ describe("ManualEntry", () => {
       fireEvent.click(screen.getByText("Continue"));
       expect(mockOnDataChange).not.toHaveBeenCalled();
 
-      expect(screen.getByText("Enter the first line of your address")).toBeInTheDocument();
-      expect(screen.getByText("Enter the town of your address")).toBeInTheDocument();
+
+      expect(
+        screen.getByText("Enter the first line of your address")
+      ).toBeInTheDocument();
+      expect(
+        screen.getByText("Enter the town of your address")
+      ).toBeInTheDocument();
+      expect(screen.getByText("Enter a real postcode")).toBeInTheDocument();
```

Comment:
Your test expects "enter a real postcode", but your component message renders "Enter your postcode", update one of these to remain consistent with the rest of the application.